google/cloud/_helpers.py

```diff
@@ -42,8 +42,10 @@
 from six.moves.http_client import HTTPConnection
 from six.moves import configparser
 
+# pylint: disable=ungrouped-imports
 from google.cloud.environment_vars import PROJECT
 from google.cloud.environment_vars import CREDENTIALS
+# pylint: enable=ungrouped-imports
```

Comment:
Sort of yeah, sort of no. I don't even know what the right way to feel about it is. Though I think we should punt on the discussion outside of this PR.